app/src/androidTest/java/com/github/h3lp3rs/h3lp/professional/VerificationActivityTest.kt

```diff
@@ -0,0 +1,99 @@
+package com.github.h3lp3rs.h3lp.professional
+
+import android.content.Intent
+import android.net.Uri
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.init
+import androidx.test.espresso.intent.Intents.release
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.h3lp3rs.h3lp.R
+import com.github.h3lp3rs.h3lp.database.Databases
+import com.github.h3lp3rs.h3lp.database.MockDatabase
+import com.google.android.gms.tasks.OnSuccessListener
+import com.google.firebase.storage.StorageReference
+import com.google.firebase.storage.UploadTask
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.kotlin.any
+import org.mockito.kotlin.doAnswer
+
+private val TEST_URI = Uri.EMPTY
+private const val TEST_STRING = ""
+private const val INTENT_TYPE = "image/*"
+
+@RunWith(AndroidJUnit4::class)
+class VerificationActivityTest {
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(
+        VerificationActivity::class.java
+    )
+
+    @Before
+    fun setup() {
+        init()
+    }
+
+    @After
+    fun clean() {
+        release()
+    }
+
+    @Test
+    fun chooseImgButtonWorks() {
+        Espresso.onView(ViewMatchers.withId(R.id.button_choose_img)).check(
```

Comment:
To be a bit more consistent with the rest of our test files, you can statically import methods like onView, intended, click, when... to avoid having IntentMatchers. or Espresso. every time